sleap/gui/commands.py

```diff
@@ -2029,6 +2033,34 @@ def ask(context: CommandContext, params: dict) -> bool:
 
         return len(filename) > 0
 
+class LinkVideoToSession(EditCommand):
+    topics = [UpdateTopic.video]
+
+    @staticmethod
+    def do_action(context: CommandContext, params: dict):
+        video = context.state["video"]
+        session = context.state["session"]
```

Comment:
The `Video` and `RecordingSession` should come from the selected items in the `UnlinkedVideosTable` and `SessionsTable` respectively. We also need the selected camera as the last piece of our linking.
```suggestion
        video = context.state["selected_unlinked_video"]
        session = context.state["selected_session"]
        camcorder = context.state["selected_camera"]
```